apps/designer/app/routes/designer/$id.tsx

```diff
@@ -44,7 +44,9 @@ export const action: ActionFunction = async ({ request, params }) => {
       };
     } catch (error) {
       if (error instanceof Error) {
-        throw new Error(error.message);
+        return {
+          error: error.message,
```

Comment:
have you tried rendering that error? I am sure its not going to work because its `errors`

Reply:
I didn't know that remix had a default for that honestly, I assumed I would have to get it manually

Reply:
done

Reply:
Showing the error now too

Reply:
nono, its just us, I am not even sure that's the best way, just got it from some examples

Reply:
all it does it returning that object into outlet component below where we handle it

Reply:
But yeah it looks good, i just didn't know we had a standard